### PR TITLE
StartSubscription not longer requires StripeToken for free plans

### PR DESCRIPTION
**Pull request by ngelx**
A minor fix to avoid the requirement of the StripeToken on StartSubscription for free plans (plans with amount == 0)

### CODE REVIEW

File: spec/dummy/db/schema.rb [hunk 1]
```diff
@@ -112,7 +112,7 @@
     t.text     "customer_address"
     t.text     "business_address"
     t.integer  "setup_fee"
-    t.decimal  "tax_percent"
+    t.integer  "tax_percent"
```
Comment (peterkeen) [inline spec/dummy/db/schema.rb]: Could you revert the changes to `db/schema.rb` please? This accidentally changes `tax_percent` back to an integer.

**Comment by ngelx** [comment]
Sorry for the delay, i will fix it today, sorry for that, not sure what happened there, I've just recreated the db and re-run the migrations to be able to test the code.

**Comment by ngelx** [comment]
@peterkeen done. Any idea why that change was filtered in first place?

**Comment by peterkeen** [comment]
@ngelx Thank you! I don't know, it's possible you accidentally added it when you rebased onto master or something.
